tests/main/remodel/task.yaml

```diff
@@ -45,6 +46,14 @@ restore: |
     systemctl start snapd.service snapd.socket
     # wait for first boot to be done
     while ! snap changes | grep -q "Done.*Initialize system state"; do sleep 1; done
+    # extra paranoia because failure to cleanup earlier took us a long time
+    # to find
+    if [ -e /var/snap/test-snapd-tools/current ]; then
+        echo "Leftover test-snapd-tools data dir found, test does not "
+        echo "properly cleanup"
+        echo "see https://github.com/snapcore/snapd/pull/6620"
```

Comment:
The URL of this branch is different, is this expected?

Reply:
Yes, this is the PR where this was first noticed and where it was first fixed. It has a bit more context about why this is done here.